Sources/FrostflakeKit/FrostflakeIdentifier+Base58.swift

```diff
@@ -0,0 +1,48 @@
+extension FrostflakeIdentifier {
+    private static let _base58Alphabet = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz"
+    private static let _base58Characters = Array(_base58Alphabet) // Convert to array for direct indexing
+    // ASCII value of 'z' is 122, so we create an array of size 123.
+    private static let _base58AlphabetIndexByChar: [Int?] = {
+        var indexes = [Int?](repeating: nil, count: 123) // 'z' is the highest ASCII character in the alphabet.
+        for (index, char) in _base58Alphabet.utf8.enumerated() {
+            indexes[Int(char)] = index
+        }
+        return indexes
+    }()
+
+    public var base58: String {
+        var number = self
+        var encodedChars: [Character] = [] // Use array to collect characters
+
+        while number > 0 {
+            let remainder = Int(number % 58)
+            number /= 58
+            encodedChars.append(Self._base58Characters[remainder]) // Append character to the array
+        }
+
+        return String(encodedChars.reversed()) // Create a string from the reversed array of characters
+    }
+
+    // Base58 Decoding
+    public init? (base58: String) {
```

Comment:
If we would throw an error then we could let caller know what is really wrong with a string.

Reply:
Well, we've had this discussion before - trivial types uses failable initialisers idiomatically... This is exactly the same behaviour as the standard UInt64 initialiser from strings, so let's follow the standard set by the standard library.